crates/shared/src/sources/balancer_v2/info_fetching.rs

```diff
@@ -1,84 +0,0 @@
-//! Responsible for conversion of a `pool_address` into `*::PoolInfo` which is
```

Comment:
Was the content of this file moved somewhere?

Reply:
No, it was completely removed.